jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/UpdateChecker.java

```diff
@@ -121,12 +121,22 @@
         if (currentVersion.equals(version.latest)) {
           return Optional.empty();
         }
-        return Optional.of(
-            "A new version of Jib ("
+        String updateMessage =
+            "A new version of "
+                + toolName
+                + " ("
                 + version.latest
                 + ") is available (currently using "
                 + currentVersion
-                + "). Update your build configuration to use the latest features and fixes!");
+                + "). ";
+        if (toolName.equals("jib-cli")) {
+          return Optional.of(
+              updateMessage
+                  + "Download the latest jib-cli version from https://github.com/GoogleContainerTools/jib/releases.");
```

Comment:
And since we have the `version.latest`, I think we can directly link the Release with the target version?

Reply:
Good point